napptive/catalog-manager
# Pg-152/user creation

### Checklist

#### What does this PR do?
implements the user creation and a cli to execute the command

#### Where should the reviewer start?

#### What is missing?

#### How should this be manually tested?
Executing:
```
$ catalog-manager user create <username> <password> -c <connectionString>
User created
```

You can check it executing:
```
$ catalog-manager user login <username> <password> -c <connectionString>
Login success
```

_The login op is incomplete, it only allows us to verify that the login is created and verify the password_

#### Any background context you want to provide?

#### What are the associated tickets?
[PG-152](https://napptive.atlassian.net/browse/PG-152)

#### Screenshots (if appropriate)

#### Questions
